test/Libraries/RevitIntegrationTests/ElevationMarkerTests.cs

```diff
@@ -18,7 +18,7 @@ namespace RevitSystemTests
     class ElevationMarkerTests : RevitSystemTestBase
     {
         
-        [Test]
+        [Test, Ignore, Category("Failure")]
```

Comment:
This test failed, in the CanCreateElevationMarker.dyn, a Node which is contained in third party package 'Rhythm', was used to get ViewFamilyType.